src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -310,9 +310,27 @@ export default defineComponent({
     })
 
     const play = () => {
-      // delay initializing the local audio element until playback is requested
+      // Delay initializing the local audio element until playback is requested
       if (!localAudio) initLocalAudio()
-      localAudio?.play()
+
+      // Check if the audio can be played successfully
+      localAudio?.play().catch((err) => {
+        let errorMsg = ''
+        switch (err.name) {
+          case 'NotAllowedError':
+            errorMsg = 'err_unallowed'
+            break
+          case 'NotSupportedError':
+            errorMsg = 'err_unsupported'
+            break
+          default:
```

Comment:
Do we want to capture this error in Sentry to figure out what it is?

Reply:
You're correct! thanks for reminding me of this.